language/tools/move-cli/src/sandbox/utils/mod.rs

```diff
@@ -345,16 +345,26 @@ pub(crate) fn explain_publish_error(
             let old_api = normalized::Module::new(&old_module);
             let new_api = normalized::Module::new(module);
 
-            if Compatibility::new(false, true, false)
-                .check(&old_api, &new_api)
-                .is_err()
+            if (Compatibility {
```

Comment:
nit: unnecessary parens?

Reply:
Needed sadly for the Rust parser to be happy :'(